modules/javafx.controls/src/main/java/javafx/scene/control/Labeled.java

```diff
@@ -818,6 +818,48 @@ public final BooleanProperty mnemonicParsingProperty() {
         return mnemonicParsing;
     }
 
+    /**
+     * Truncated read-only property indicates whether the text has been truncated
+     * when it cannot fit into the available width.
```

Comment:
No need to repeat the name and type in the first sentence as it's used in the summary table. Most phrasings opt (inconsistently for): "Indicates whether..." or just "Whether..." (I prefer the former).

I'm also not sure about the sole role of the width. Can the text not be truncated for a reason other than not fitting in the width, like not fitting in the height?